test/common/scale_up_replicas_apicurito.go

```diff
@@ -68,26 +106,27 @@ func getApicurito(dynClient k8sclient.Client) (apicuritov1alpha1.Apicurito, erro
 	return *apicuritoCR, nil
 }
 
-func updateApicurito(dynClient k8sclient.Client, replicas int32, t *testing.T) (apicuritov1alpha1.Apicurito, error) {
+func updateApicurito(dynClient *TestingContext, replicas int32, t *testing.T) (apicuritov1alpha1.Apicurito, error) {
```

Comment:
```suggestion
func updateApicurito(ctx *TestingContext, replicas int32, t *testing.T) (apicuritov1alpha1.Apicurito, error) {
```

Small name change but keep it consistent with other functions.